### PR TITLE
bugfix/LIVE-3322-back-to-edit-fees

### PR DESCRIPTION
**Pull request by henrily-ledger**
### 📝 Description

Change in modal sign transaction to make more visible the changeFees

### ❓ Context

- **Impacted projects**: `live-desktop` 
- **Linked resource(s)**: `https://ledgerhq.atlassian.net/browse/LIVE-3322` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

Screenshot 2022-08-11 at 14 05 41

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/components/Modal/ModalHeader.jsx [hunk 1]
```diff
@@ -106,16 +107,22 @@ const ModalHeader = ({
   onBack?: void => void,
   onClose?: void => void,
   style?: *,
+  backButtonComponent?: React$Node,
 }) => {
   const { t } = useTranslation();
+  console.log({backButtonComponent})
```
Comment (hedi-edelbloute) [inline apps/ledger-live-desktop/src/renderer/components/Modal/ModalHeader.jsx:113]: upsie :smile:

File: apps/ledger-live-desktop/src/renderer/modals/SignTransaction/Body.jsx [hunk 2]
```diff
@@ -221,6 +229,8 @@ export default function Body({ onChangeStepId, onClose, setError, stepId, params
 
   if (!status) return null;
 
+  console.log(steps);
```
Comment (hedi-edelbloute) [inline apps/ledger-live-desktop/src/renderer/modals/SignTransaction/Body.jsx:232]: :cold_sweat:

**Review by hedi-edelbloute — APPROVED** [review]

**Comment by henrily-ledger** [comment]
Screenshot 2022-08-11 at 16 30 16

**Pull request merged**